.travis.yml

```diff
@@ -4,6 +4,14 @@ rust:
 branches:
   only:
     - bench
+before_install:
+  - curl -L https://downloads.sourceforge.net/project/gnuplot/gnuplot/5.0.6/gnuplot-5.0.6.tar.gz > gnuplot-5.0.6.tar.gz
+  - tar xzf gnuplot-5.0.6.tar.gz
+  - cd gnuplot-5.0.6
+  - ./configure
+  - make && make install
```

Comment:
Not entirely sure if `gnuplot` has an `apt` package. I'll try to debug it a bit, but I'm somewhat limited right now since I can't get the travis cli set up on my computer.